cmd/create/oidcconfig/cmd.go

```diff
@@ -265,7 +253,15 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
-	oidcConfigInput := buildOidcConfigInput(r)
+	oidcConfigInput := oidc_config.OidcConfigInput{}
+	if !args.managed {
```

Comment:
Any reason to call this explicitly instead of letting the builder handle it?

Reply:
In my idea the caller should be aware if the oidc config is empty or not (due to the managed attribute)

Reply:
It's ok too I suppose